src/command.cc

```diff
@@ -988,7 +1019,9 @@ void Run(TextBox& tbox) {
       tbox.putc('\n');
       return;
     }
-    Process& proc = LoadELFAndCreateEphemeralProcess(*file);
+    char* pname = new char[128 + 1];
```

Comment:
pname should be dynamically sized because there is no guarantee that the buf has efficient space to store the string passed. You can do something like: `char* pname = new char[strlen(line + 1)];`